docs/modules/user-guide/partials/ref_devfile-samples.adoc

```diff
@@ -3,5 +3,8 @@
 
 .Additional resources
 
-* link:https://github.com/devfile/api/issues[Creating devfiles].
-* link:https://github.com/devfile/docs[Documenting devfiles].
+To view the source code for the devfile specification, visit the link:https://github.com/devfile/api[GitHub API repository].
+
+To track the creation of devfiles, review the link:https://github.com/devfile/api/issues[open issues].
```

Comment:
I don't like how this line is phrased, it's not really clear what "tracking the creation of devfiles" means

I think this line should be changed to say something like "To track the development of the devfile specification, view our issue tracker at ..." (or phrased similarly).

Reply:
+1